ide/che-core-ide-api/src/main/java/org/eclipse/che/ide/api/editor/gutter/Gutters.java

```diff
@@ -17,6 +17,9 @@ private Gutters() {}
   /** Logical identifer for the breakpoints gutter. */
   public static final String BREAKPOINTS_GUTTER = "breakpoints";
 
+  /** Logical identifer for the vcs change markers gutter. */
+  public static final String VCS_CHANGE_MARKERS_GUTTER = "vcsChangeMarkersGutter";
```

Comment:
`VCS_GUTTER` ? Are there more gutters for vcs?

Reply:
I think `VCS_GUTTER` is not clear enouth, but `VCS_CHANGE_MARKERS_GUTTER` describes what can be shown on the gutter.